frontstage_api/controllers/case_controller.py

```diff
@@ -126,17 +126,17 @@ def build_full_case_data(case):
 
 def calculate_case_status(case):
     logger.debug('Getting the status of case')
-    case_events = case.get('caseEvents')
-    status = None
-    if case_events:
-        for event in case_events:
-            if event['category'] == 'SUCCESSFUL_RESPONSE_UPLOAD':
-                status = 'Complete'
-                break
-            elif event['category'] == 'COLLECTION_INSTRUMENT_DOWNLOADED':
-                status = 'Downloaded'
+    case_group_status = case.get('caseGroup', {}).get('caseGroupStatus')
+
+    if case_group_status == 'COMPLETE':
+        status = 'Complete'
+    elif case_group_status == 'INPROGRESS':
+        status = 'Downloaded'
```

Comment:
Status will have a default value of "NOTSTARTED" defined at the schema level

Reply:
status will be set to 'Not started' as default